hello-world/hello-world-theme/home.hubl.html

```diff
@@ -11,4 +11,11 @@
     path="@projects/hello-world-project/hello-world-app/components/partials/Home.jsx",
     no_wrapper=True %}
 
+  {% dnd_area "dnd_area", class="main" %}
+    {% dnd_section %}
+      {% dnd_module path="@projects/hello-world-project/hello-world-app/components/modules/TodoList" %}
+      {% end_dnd_module %}
+    {% end_dnd_section %}
+  {% end_dnd_area %}
+
```

Comment:
I think it would be a little better to add `dnd_module path="@projects/hello-world-project/hello-world-app/components/modules/TodoList"` to https://github.com/HubSpot/cms-js-building-block-examples/blob/main/hello-world/hello-world-theme/todo.hubl.html instead of home.hubl? (Maybe leave this D&D area in home as empty though?)